recipes/avro-python3/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "avro-python3" %}
+{% set version = "1.10.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/avro-python3-{{ version }}.tar.gz
+  sha256: 3b63f24e6b04368c3e4a6f923f484be0230d821aad65ac36108edbff29e9aaab
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - avro
+    - isort
+    - pip
+    - pycodestyle
```

Comment:
Are `isort` and `pycodestyle` really needed to build the package?

Reply:
Sadly yes. Some people don't know how to package dev deps properly :-(